src/transactions/NoActivity.tsx

```diff
@@ -7,8 +7,8 @@ import { withTranslation } from 'src/i18n'
 import colors from 'src/styles/colors'
 import { typeScale } from 'src/styles/fonts'
 interface OwnProps {
-  loading: boolean
-  error: Error | FetchBaseQueryError | SerializedError | undefined
+  loading?: boolean
+  error?: Error | FetchBaseQueryError | SerializedError | undefined
```

Comment:
i would remove these props but this is currently being used by the old transaction feed and i don't want to spend time on that component